tools/wptserve/tests/functional/base.py

```diff
@@ -75,7 +75,7 @@ def request(self, path, query=None, method="GET", headers=None, body=None, auth=
             req.add_data(body)
 
         if auth is not None:
-            req.add_header("Authorization", "Basic %s" % base64.b64encode('%s:%s' % auth))
+            req.add_header("Authorization", b"Basic %s" % base64.b64encode(("%s:%s" % auth).encode("utf-8")))
```

Comment:
I wonder if we want to not encode there, so we can test with non-UTF-8 values?

Reply:
Done. Also added a test case (in Shift JIS).